src/Stryker.Core/Stryker.Core/Initialisation/InitialBuildProcess.cs

```diff
@@ -22,10 +22,22 @@ public InitialBuildProcess(IProcessExecutor processExecutor = null)
             _logger = ApplicationLogging.LoggerFactory.CreateLogger<InitialBuildProcess>();
         }
 
-        public void InitialBuild(string path, string projectName)
+        public void InitialBuild(bool fullFramework, string path, string projectName)
         {
             _logger.LogInformation("Starting initial build");
-            var result = _processExecutor.Start(path, "dotnet", $"build {projectName}");
+            ProcessResult result = null;
+            if (fullFramework)
+            {
+                // Build with MSBuild.exe
+                var msBuildPath = new MsBuildHelper().GetMsBuildPath();
+                result = _processExecutor.Start(path, msBuildPath, "");
```

Comment:
If there are two csproj files in the directory, this will fail. We should pass the csproj name to msbuild as well.